turbinia/task_manager.py

```diff
@@ -123,11 +128,23 @@ def add_evidence(self, evidence_):
     log.info('Adding new evidence: {0:s}'.format(str(evidence_)))
     self.evidence.append(evidence_)
     job_count = 0
+    jobs_whitelist = evidence_.config.get('jobs_whitelist', [])
+    jobs_blacklist = evidence_.config.get('jobs_blacklist', [])
+    jobs_list = self.jobs
+    if jobs_whitelist and jobs_blacklist:
```

Comment:
This is already checked and logged in get_jobs(), remove this one?

Reply:
Check removed.